src/Entity/OgRole.php

```diff
@@ -165,4 +200,123 @@ public function save() {
     parent::save();
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function set($property_name, $value) {
+    // Prevent the ID, role type, entity type or bundle to be changed for any of
+    // the default roles. These default roles are required and shouldn't be
+    // tampered with.
+    $is_locked_property = in_array($property_name, [
+      'id',
+      'role_type',
+      'group_type',
+      'group_bundle',
```

Comment:
`group_id` should also be part of this (for cases where a group overrides the global roles and permissions and uses it's own roles)